pages.ar/linux/dockerd.md

```diff
@@ -0,0 +1,24 @@
+# dockerd
+
+> هي عملية مستمرة تعمل في الخلفية تبدأها لتتحكم في حاويات الدوكر.
+> لمزيد من التفاصيل: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
```

Comment:
I don't speak Arabic but is this a better translation from `more information`?
```suggestion
> معلومات اكثر: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
```

Reply:
Its meaning the same as the first, but the first is the more standard.